modules/javafx.graphics/src/main/java/com/sun/glass/ui/gtk/GtkApplication.java

```diff
@@ -51,72 +50,32 @@
 import java.util.concurrent.CountDownLatch;
 import java.lang.annotation.Native;
 
+
 final class GtkApplication extends Application implements
                                     InvokeLaterDispatcher.InvokeLaterSubmitter {
-    private static final String SWT_INTERNAL_CLASS =
-            "org.eclipse.swt.internal.gtk.OS";
     private static final int forcedGtkVersion;
     private static boolean gtk2WarningIssued = false;
-    private static final String GTK2_ALREADY_LOADED_WARNING =
-        "WARNING: Found GTK 2 library already loaded";
     private static final String GTK2_SPECIFIED_WARNING =
-        "WARNING: A command line option has enabled the GTK 2 library";
-    private static final String GTK2_FALLBACK_WARNING =
-        "WARNING: Using GTK 2 library because GTK 3 cannot be loaded ";
-    private static final String GTK2_DEPRECATION_WARNING =
-        "WARNING: The JavaFX GTK 2 library is deprecated and will be removed in a future release";
+        "WARNING: A command line option tried to enable the GTK 2 library";
 
+    private static final String GTK2_REMOVED_WARNING =
+        "WARNING: The JavaFX GTK 2 library was removed.";
```

Comment:
I've inserted it, but opted for a new sentence.

Reply:
I would like to see an additional info message, e.g. "continuing using GTK 3 instead" or similar. It's already good that the new warning says that the provided option will be ignored, but the user doesn't know the consequences.
I do realize it is getting a bit verbose, but as a user, I think there is value in seeing the severity of a warning. Hence, if we can avoid leaving the user with an "Ok, so what now?" feeling, I would vote for that.

I would even repeat the exact command line option that caused this, so that users can look where this option was set.